packages/mdc-tab-indicator/_mixins.scss

```diff
@@ -30,6 +30,7 @@
 @mixin mdc-tab-indicator-underline-color($color) {
   > .mdc-tab-indicator__content--underline {
     @include mdc-theme-prop(background-color, $color);
```

Comment:
I think you could remove this background-color as well. I think it's a good opportunity to add a screenshot test to confirm that we didn't change anything. I can help you with that. :)

Reply:
Moved to using just border to make it consistent between high-contrast mode and normal.